src/integration-tests/introspector/introspectTest.sh

```diff
@@ -123,7 +138,7 @@ function cleanupMajor() {
 
   rm -fr /tmp/introspect
   mkdir -p $test_home || exit 1
-
+  rm -fr ${SCRIPTPATH}/mii/workdir
```

Comment:
This should use ${test_home}/mii/workdir throughout - same location as the rest of the tests temp files...

Reply:
changed